exporter/awscloudwatchlogsexporter/utils.go

```diff
@@ -0,0 +1,45 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package awscloudwatchlogsexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awscloudwatchlogsexporter"
+
+import (
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/cwlogs"
+)
+
+func attrsString(attrs map[string]interface{}) map[string]string {
+	out := make(map[string]string, len(attrs))
+	for k, s := range attrs {
+		out[k] = s.(string)
```

Comment:
This is risky here since attributes can be of other types besides string, if you always want the string value here then it would be better to do:

```
  out[k] = fmt.Sprint(s)
```